css-backgrounds-4/Overview.bs

```diff
@@ -771,16 +771,28 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either 'none' or a list,
-	each item a pair of offsets (horizontal and vertical) from the element‘s box
-Animatable: by computed value
+Computed value: list, each item either 'none' or a pair of offsets
+  (horizontal and vertical) from the element‘s box
+Animatable: by computed value,
+    appending blank shadows (''transparent 0 0 0 0'')
+    with a corresponding ''box-shadow-position/inset'' keyword as needed
+    to match the longer list
+    if the shorter list is otherwise compatible with the longer one
```

Comment:
I guess that this should actually be split and put the relevant part to the corresponding longhand.
So `box-shadow-offset` will append `0 0`, `box-shadow-color` will append `transparent`, etc.

Also, to preserve the old behavior, I guess that `none` should be treated as `0 0` when interpolated with a non-`none` value?